src/sentry/integrations/utils/codecov.py

```diff
@@ -0,0 +1,31 @@
+from typing import Optional, Sequence, Tuple
```

Comment:
When adding new files check to see if the file is listed under mypy.ini

Reply:
Thank you, this is good to know!